manifests/plugin.pp

```diff
@@ -43,7 +43,7 @@
   $source = undef,
   $ensure = present,
   $environment = [],
-  String $user = $logstash::logstash_user,
+  String $user = 'root',
```

Comment:
Package installs logstash as root and logstash-plugin command needs access to /usr/share/logstash/Gemfile

Reply:
this is a bit tricky. will this break existing installations or did it never work?

Reply:
We don't use any plugins and I really don't know logstash that well.
But I do not see how this could ever have worked.
The default $logstash::logstash_user is not created by the module, its created by the package.
The package installs logstash at /usr/share/logstash with root as owner.
$logstash_user is only used to run the service.

Even if someone has a profile class that changes the ownership of /usr/share/logstash and they don't override the $user param, new plugins will be installed with root as owner  but I do think logstash running as a less privileged user will still have read access to the installed plugin.

Might be a good idea to mention this in the release notes though.

Reply:
@bastelfreak have a look at #413

Reply:
@thebeanogamer do you have any input here?